sale_service_project/README.rst

```diff
@@ -0,0 +1,72 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+====================
+Sale Service Project
+====================
+
+This module adds the option to assign materials and/or work to a service
+product type which serve as a template transferring this information to the
+line order which may be changed.
+This module extends the functionality of the *sale_service* module creating a
+new project for each order adding materials and works informed on the order
+line to the generated task.
+
+Usage
+=====
+
+To use this module, you need to:
+
+1. Create a service product and check 'Create Task Automatically'.
```

Comment:
Go to Sales -> Products and create a product with type Service, check also option 'Create task automatically'